build/makelib/cephd.mk

```diff
@@ -74,9 +79,6 @@ cephd.config:
 .PHONY: cephd.build
 cephd.build: cephd.config
 	@cd $(CEPHD_BUILD_DIR)/$(CEPHD_PLATFORM) && $(MAKE) cephd
-ifeq ($(CEPHD_DEBUG),0)
-	@$(STRIP) -pd $(CEPHD_BUILD_DIR)/$(CEPHD_PLATFORM)/lib/libcephd.a
```

Comment:
we need the debug information all intact, and we move it into the .debug file after linking